astro.config.ts

```diff
@@ -416,11 +416,6 @@ export default defineConfig({
 								uk: "Розширення Zed",
 							},
 						},
-						{
-							label: "GritQL",
-							link: "/reference/gritql",
-							badge: "experimental",
-						},
```

Comment:
Why was this removed, along with the entire `gritql.mdx` file? That was already part of 1.9.

Reply:
Sorry, was thought it is not available yet. I'll revert it asap

Reply:
Reverted. Thank you for noticing my mistake 🙇

Reply:
No worries! Thanks for the effort!